### PR TITLE
Components/wizard

### PR DESCRIPTION
**Pull request by josiahhaswell**

### CODE REVIEW

File: aire-ui/src/main/java/io/sunshower/zephyr/ui/components/Card.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import com.vaadin.flow.component.html.Section;
+import lombok.NonNull;
+import lombok.val;
+
+@Tag("aire-card")
+@JsModule("./aire/ui/components/card.ts")
+@CssImport("./styles/aire/ui/components/card.css")
+public class Card extends HtmlContainer implements ClickNotifier<Card> {
+
+  public static final String SLOT = "slot";
```
Comment (lisagumerman) [inline aire-ui/src/main/java/io/sunshower/zephyr/ui/components/Card.java:21]: We use these strings a lot throughout -- worth pulling into a separate set of constants?

File: aire-ui/src/main/java/io/sunshower/zephyr/ui/components/Wizard.java [hunk 2]
```diff
@@ -0,0 +1,651 @@
...
+      throw new IllegalArgumentException(
+          format(
+              "Error: state-key '%s' is not registered--please register it via 'addStep' before calling this method",
+              key));
+    }
+    currentStep = step;
+  }
+
+  /**
+   * add a transition from <code>from</code> to <code>to</code>
```
Comment (lisagumerman) [inline aire-ui/src/main/java/io/sunshower/zephyr/ui/components/Wizard.java:115]: My brain broke for a second there

File: aire-ui/src/main/java/servlet/ZephyrModuleResourceServlet.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  private final Kernel kernel;
+  private final Module rootModule;
+
+  public ZephyrModuleResourceServlet(final Kernel kernel, final Module rootModule) {
+    this.kernel = kernel;
+    this.rootModule = rootModule;
+  }
+
+  @Override
+  @SuppressWarnings(value = "PMD.UseProperClassLoader")
```
Comment (lisagumerman) [inline aire-ui/src/main/java/servlet/ZephyrModuleResourceServlet.java:31]: Improper class loaders only

**Review by lisagumerman — APPROVED** [review]

File: aire-ui/src/main/java/io/sunshower/zephyr/ui/components/Card.java (hunk 1 above)
Comment (josiahhaswell) [reply aire-ui/src/main/java/io/sunshower/zephyr/ui/components/Card.java:21]: I hear what you're saying, but after some thought, I think that colocating styles has some disadvantages:

1. It removes the association between a style and its host component
2. Changing the style could break multiple components

**Pull request merged**